sources/packages/web/src/views/student/StudentApplicationSummary.vue

```diff
@@ -105,6 +94,13 @@ export default {
       reloadData.value = true;
     };
 
+    const confirmCancelApplication = async (applicationId: number) => {
+      if (await cancelApplicationModal.value.showModal(applicationId)) {
+        // Reload details.
+        setReloadData();
```

Comment:
Can we try to get rid of this workaround?
One option would be the event to pass the `applicationId`, as it is today, and one more method to be invoked to refresh the list. Does it make sense?

Reply:
okay. I thought the purpose of `showModal` param was to pass params 
![image](https://user-images.githubusercontent.com/77353155/195957708-d32011c4-90c9-46e7-80ba-fde55753faa8.png)

Reply:
I am writing here just to update the other devs about our conversation 😉 
The `showModal` can receive parameters but that was not the workaround that I was talking about. The idea was to get rid of the `setReloadData` which is a workaround to reload the data in the other component. The implementation that we reached during our call (the one that will replace this workaround) seems better, as we talked.

Reply:
I kinda understand what is happening here. But what was the concluded solution?

Reply:
The solution is the one present in the PR. The event receives a reference for the method that can be called to update the summary list.
![image](https://user-images.githubusercontent.com/61259237/196274296-23b6e588-b120-4da5-b043-289061cb5a9f.png)